packages/notebook-extension/src/index.ts

```diff
@@ -1369,8 +1369,20 @@ function activateNotebookHandler(
       ).composite as boolean,
       renderingLayout: settings.get('renderingLayout').composite as
         | 'default'
-        | 'side-by-side'
+        | 'side-by-side',
+      sideBySideLeftMarginOverride: settings.get('sideBySideLeftMarginOverride')
+        .composite as string,
+      sideBySideRightMarginOverride: settings.get(
+        'sideBySideRightMarginOverride'
+      ).composite as string
     };
+    document.head.insertAdjacentHTML(
+      'beforeend',
+      `<style>.jp-mod-sideBySide.jp-Notebook .jp-Notebook-cell { 
+      margin-left: ${factory.notebookConfig.sideBySideLeftMarginOverride} !important;
+      margin-right: ${factory.notebookConfig.sideBySideRightMarginOverride} !important;
+    }</style>`
+    );
```

Comment:
What about giving a unique id to the `style` tag, so that you can retrieve it and update it when the settings change?

Reply:
Thx for the review @fcollonval The last commit implements your suggestion.